transformations/space_between_characters/transformation.py

```diff
@@ -0,0 +1,44 @@
+import random
+from typing import List
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+def add_spaces(text, prob=0.1, seed=0, max_outputs=1):
+    random.seed(seed)
+
+    words = text.split(" ")
+    perturbed_texts = []
+    for _ in range(max_outputs):
+        perturbed_text = []
+        for word in words:
+            if random.random() <= prob:
+                new_word = " ".join(word)
+            else:
+                new_word = word
+            perturbed_text.append(new_word)
+        perturbed_texts.append(" ".join(perturbed_text))
+    return perturbed_texts
+
+
+class SpaceBetweenCharacters(SentenceOperation):
+    tasks = [
+        TaskType.TEXT_CLASSIFICATION,
+        TaskType.TEXT_TO_TEXT_GENERATION,
+        TaskType.TEXT_TAGGING,
+    ]
+    languages = ["en"]
```

Comment:
By using another tokenizer, this could also work for other languages. The "en" choice is surprising here.

Reply:
You are right, thank you for spotting this. I have changed it to "all" in 28b1301c23bd7105017b274a64501f32328309f1